### ISSUE TITLE
Dont build release version

### ISSUE THREAD

**Issue by Skyline404** [rusty-bits/OC-tool#28]
```
iMac-Andrey:OC-tool andrey$ ./OC-tool 
To build from source use -s option
For help use './OC-tool -h'

Checking OC-tool for updates ... done

Processing config.plist ... done
Checking for missing fields ... done
Checking for extra fields ... done
Using /Volumes/Data/Andrey/OC-tool/INPUT/config.plist

Downloading/unzipping needed or updated resources
Downloading  ... iMac-Andrey:OC-tool andrey$ 
```

**Comment by rusty-bits** [comment]
It looks like OC-tool is messing up and reading something wrong from the plist, then it's trying to download a "blank"  

can you share your plist (without serial # ids ) so I can see where OC-tool is choking?  
I probably messed something up in the git_repo.json file  

also, can you run `./OC-tool ./Docs/Sample057.plist` and make sure that builds so we know things are okay on your end?
and maybe also let me know what you see in the log, it might help me fix what I messed up  
`cat tool.log`

**Comment by rusty-bits** [comment]
I could not reproduce the problem you were having, but I did find an issue caused by the new ReservedMemory section in the config.plist  

OC-tool was not saving the config.plist correctly if the last section in the plist was an array, this has been fixed in commit fd48bb6  

If you still have an issue please reopen this ticket, or create a new one with additional info
